### PR TITLE
Remove Drag&Drop - #326

### PR DESCRIPTION
**Pull request by BakhromSadullaev**

### CODE REVIEW

File: src/App/components/Section/components/LocationBlock/LocationBlock.tsx [hunk 1]
```diff
@@ -185,19 +141,13 @@ const LocationBlock: React.FC<ILocationBlockProps> = ({
           [style.onboarding]:
             !index && (onboarding?.dragDropMode || onboarding?.myLocation || onboarding?.comment)
```
Comment (shokhrukh-epam) [inline src/App/components/Section/components/LocationBlock/LocationBlock.tsx:142]: delete onboarding?.drapDropMode

File: src/App/components/Header/components/SettingsModal/SettingsModal.tsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ const SettingsModal: React.FC = () => {
   const { autoTheme, theme, showDate, showCountry, showTimezone, timeFormat, autoSorting } =
```
Comment (shokhrukh-epam) [inline src/App/components/Header/components/SettingsModal/SettingsModal.tsx:33]: delete autoSorting

File: src/App/redux/reducer.ts [hunk 3]
```diff
@@ -12,9 +12,6 @@ const initialState: IInitialState = {
   deleteMode: {
     isOn: false
   },
-  dragDropMode: {
-    isOn: false
-  },
   settings: {
     theme: THEME.light,
     autoTheme: undefined,
```
Comment (shokhrukh-epam) [inline src/App/redux/reducer.ts:17]: delete autosorting

File: src/App/hooks/useOnboarding.tsx [hunk 4]
```diff
@@ -31,17 +29,17 @@ const useOnboarding = () => {
   };
 
   let localSettings = localStorage.getItem('settings');
-  let parsedSettings: any, isAutoSortingOn: boolean;
+  let parsedSettings: any;
 
   if (localSettings) {
     parsedSettings = JSON.parse(localSettings);
-    isAutoSortingOn = parsedSettings['autoSorting'];
+    // isAutoSortingOn = parsedSettings['autoSorting'];
```
Comment (ArturSalikhovEPAM) [inline src/App/hooks/useOnboarding.tsx:36]: do we need it?

File: src/App/components/Section/components/LocationBlock/LocationBlock.tsx [hunk 5]
```diff
@@ -27,177 +22,45 @@ const LocationBlock: React.FC<ILocationBlockProps> = ({
   setSelectedLocation,
```
Comment (ArturSalikhovEPAM) [inline src/App/components/Section/components/LocationBlock/LocationBlock.tsx:22]: selectedLocation and setSelectedLocation - do we need this props?

File: src/App/components/Section/components/LocationBlock/components/CommentButton/CommentButton.tsx [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+    setLocations(locations);
+    handleCloseCommentModal();
+  };
+
+  useEffect(() => {
+    if (location && locations[location.city + location.lat].comment) {
+      setInputText(locations[location.city + location.lat].comment || '');
+    }
+    // use it only when component mount
+    // eslint-disable-next-line
```
Comment (ArturSalikhovEPAM) [inline src/App/components/Section/components/LocationBlock/components/CommentButton/CommentButton.tsx:56]: do we still need this comment?

File: src/App/components/Section/components/LocationBlock/components/CommentButton/CommentButton.module.scss [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+    font-family: roboto-bold;
+    font-size: $fs20;
+    margin-bottom: $space-l;
+  }
+
+  .input {
+    width: 100%;
+    border: 1px solid $blueColor;
+    border-radius: 5px;
+    height: 50px;
```
Comment (ArturSalikhovEPAM) [inline src/App/components/Section/components/LocationBlock/components/CommentButton/CommentButton.module.scss:17]: can we change here scss variables?

File: src/App/components/Section/components/LocationBlock/components/TimeInfo/TimeInfo.tsx [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+  const { counter, planningMode } = useSelector((state: IInitialState) => state);
+  const { showDate, showTimezone, timeFormat } = useSelector(
+    (state: IInitialState) => state.settings
+  );
+  const { userLocation } = useSelector((state: IInitialState) => state.locations);
+
+  useEffect(() => {
+    setTime(timeInfo);
+    // don't need as a dependency timeInfo
+    // eslint-disable-next-line
```
Comment (ArturSalikhovEPAM) [inline src/App/components/Section/components/LocationBlock/components/TimeInfo/TimeInfo.tsx:35]: please double check do ewe still need this comment here

**Comment by ArturSalikhovEPAM** [comment]
@BakhromSadullaev also please check en.json file and delete fields that we won't be use anymore (Drag&Drop)

File: src/App/components/Section/components/LocationBlock/components/TimeInfo/TimeInfo.tsx (hunk 8 above)
Comment (BakhromSadullaev) [reply src/App/components/Section/components/LocationBlock/components/TimeInfo/TimeInfo.tsx:35]: ESlint configuration is needed

**Review by ArturSalikhovEPAM — APPROVED** [review]
LGTM!

**Review by shokhrukh-epam — APPROVED** [review]
LGTM!

**Pull request merged**